src/pages/publication/submission_tab/submission_journal.tsx

```diff
@@ -3,27 +3,26 @@ import React from 'react';
 import { useTranslation } from 'react-i18next';
 import { FormikProps, useFormikContext } from 'formik';
 import { Publication } from '../../../types/publication.types';
-import FormCardSubHeading from '../../../components/FormCard/FormCardSubHeading';
 
 const SubmissionJournalPublication: React.FC = () => {
   const { t } = useTranslation('publication');
   const { values }: FormikProps<Publication> = useFormikContext();
 
   return (
     <>
-      <FormCardSubHeading>{t('references.journal_publication')}</FormCardSubHeading>
-
-      <LabelContentRow label={t('common:type')}>{values.reference.journalArticle?.type}</LabelContentRow>
+      <LabelContentRow label={t('common:type')}>{t('referenceTypes:journalArticle')}</LabelContentRow>
+      <LabelContentRow label={t('references.subtype')}>
+        {t(`referenceTypes:subtypes_journal_article.${values.reference.journalArticle?.type}`)}
```

Comment:
frykter litt at vi kommer til å få noen problemer når vi har såpass mange dynamiske i18n keys hvor siste verdi kan være null, men det må vi sikkert bare leve med :/

Reply:
Vi kunne jo laget en funksjon som sjekker for null, men de verdien er rimelig statiske.